environment/src/main/java/com/sequenceiq/environment/environment/service/EnvironmentVerticalScaleService.java

```diff
@@ -56,7 +56,8 @@ private void validateEntitlements(EnvironmentDto environment) {
     private void validateUpgrade(EnvironmentDto environment) {
         if (!environment.getStatus().isVerticallyScalablePhase()) {
             throw new BadRequestException(
-                String.format("Environment '%s' must be STOPPED to start Vertical Scaling.", environment.getName()));
+                String.format("Environment '%s' must be STOPPED to start Vertical Scaling. The environment state is %s", environment.getName(),
```

Comment:
maybe the STOPPED is invalid in the message @doktoric

Reply:
I think this was reverted or not ?